src/libos/crates/host-socket/src/lib.rs

```diff
@@ -13,9 +13,13 @@ extern crate sgx_tstd as std;
 
 #[macro_use]
 mod prelude;
+mod common;
 mod runtime;
 mod stream;
 mod util;
 
+pub use self::common::sockopt::{
+    new_getsockopt_cmd, new_setsockopt_cmd, SoAcceptConnCmd, SoDomainCmd, SoPeerNameCmd, SoTypeCmd,
+};
```

Comment:
I think sockopt-related features do not qualify as the top-level APIs. Making the `sockopt` module public and exposing them beneath the module is better.